src/applications/vaos/express-care/redux/reducer.js

```diff
@@ -104,11 +105,25 @@ export default function expressCareReducer(state = initialState, action) {
       };
     }
     case FORM_PAGE_CHANGE_COMPLETED: {
+      let previousPages = state.newRequest.previousPages;
+      if (!Object.keys(previousPages).length) {
+        previousPages = {
+          ...previousPages,
+          [action.pageKey]: 'home',
+        };
+      }
+      if (action.direction === 'next') {
+        previousPages = {
+          ...previousPages,
+          [action.pageKeyNext]: action.pageKey,
+        };
+      }
       return {
         ...state,
         newRequest: {
           ...state.newRequest,
           pageChangeInProgress: false,
+          previousPages: state.newRequest.previousPages,
         },
```

Comment:
Looks like you're passing the unmodified `state.newRequest.previousPages` here.  You should instead pass the updated `previousPages`.  e.g.

```js
return {
  ...state,
  newRequest: {
    ...state.newRequest,
    pageChangeInProgress: false,
    previousPages,
  },
};
```

Just another suggestion, but on line 108, maybe rename the new `previousPages` to something like `updatedPreviousPages`, then return that in the updated state on line 126.  Might be a little clearer.